controllers/signup.js

```diff
@@ -0,0 +1,33 @@
+const User = require('../models/user');
+const signupValidate = require('../auth/signup-auth')
```

Comment:
use a code formatter, it helps to format the codebase...
add missing semi colons